README.md

```diff
@@ -58,30 +58,29 @@ Next, step through the following commands to setup secure and private communicat
 # This will create a Space and Project for you in Ockam Orchestrator and provision an
 # End-to-End Encrypted Cloud Relay service in your `default` project at `/project/default`.
 ockam enroll
-ockam project information --output json > default-project.json
 
 # -- APPLICATION SERVICE --
 
-# Start an application service, listening on a local ip and port, that clients would access
-# through the cloud encrypted relay. We'll use a simple http server for this first example
+# Start an application service, listening on a local IP and port, that clients would access
+# through the cloud encrypted relay. We'll use a simple HTTP server for this first example
 # but this could be any other application service.
 python3 -m http.server --bind 127.0.0.1 5000
```

Comment:
Let's change the port here to 6000, we know that 5000 conflicts on macos sometimes.

Reply:
Done 👍